front50-protobuf/src/main/proto/com/netflix/spinnaker/front50/proto/plugins.proto

```diff
@@ -0,0 +1,48 @@
+syntax = "proto3";
+
+option java_package = "com.netflix.spinnaker.front50.proto";
+option java_multiple_files = true;
+
+message Plugin {
```

Comment:
Need more docs on both the message-level and the individual property level. For example, `version` - this should be known that it adheres to semver.org.